src/main/java/JigaChat.java

```diff
@@ -0,0 +1,153 @@
+        String[] commands = new String[2];
+        Scanner in = new Scanner(System.in);
+        input = in.nextLine();
+        commands = input.split(" ", 2);
+
+        switch(commands[0]) {
+        case "help":
+            printCommandList();
+            return;
+        case "bye":
+            System.out.println("Bye. Hope to see you again soon!");
+            System.exit(0);
+        case "list":
+            printList();
+            return;
+        case "mark":
+            try {
+                taskList[Integer.parseInt(commands[1]) - 1].markAsDone();
+                System.out.println("JigaChat has marked task " + (Integer.parseInt(commands[1])) + " as done!");
+            }
+            catch(NullPointerException e) {
+                System.out.println(commands[1] + " is not in your list!");
+            }
+            catch(ArrayIndexOutOfBoundsException e) {
+                printMarkCommand();
+            }
+            return;
+        case "unmark":
+            try {
+                taskList[Integer.parseInt(commands[1]) - 1].markAsUndone();
+                System.out.println("JigaChat has marked task " + (Integer.parseInt(commands[1])) + " as not done!");
+            }
+            catch(NullPointerException e) {
+                System.out.println(commands[1] + " is not in your list!");
+            }
+            catch(ArrayIndexOutOfBoundsException e) {
+                printUnmarkCommand();
+            }
+            return;
+        }
+        try {
+            addToList(commands);
+        }
+        catch (InvalidCommandException e) {
+            System.out.println("Command " + input + " is not recognised!");
+        }
+        catch (ArrayIndexOutOfBoundsException e) {
+            System.out.println("Type [help] to learn how to add tasks");
+        }
+        catch (StringIndexOutOfBoundsException e) {
+            System.out.println("Type [help] to learn how to add tasks");
+        }
+    }
+
+    public void addToList(String[] taskToAdd) throws InvalidCommandException {
+            if (taskToAdd[0].equals("todo")) {
+                    taskList[taskCounter] = new ToDo(taskToAdd[1]);
+            }
+            else if (taskToAdd[0].equals("deadline")) {
+                String[] deadline = taskToAdd[1].split("/", 2);
+                String by = deadline[1].substring(3);
+                String description = deadline[0].substring(0, deadline[0].length() - 1);
+                taskList[taskCounter] = new Deadline(description, by);
+            }
+            else if (taskToAdd[0].equals("event")) {
+                String[] event = taskToAdd[1].split("/", 3);
+                String description = event[0].substring(0, event[0].length() - 1);
+                String from = event[1].substring(5, event[1].length() - 1);
+                String to = event[2].substring(3);
+                taskList[taskCounter] = new Event(description, from, to);
+            }
+            else {
```

Comment:
if.. else blocks should be in this form:

```java
if (...) {
    //...
} else if (...) {
    //...
} else {
    //...
}
```

https://se-education.org/guides/conventions/java/basic.html#layout